### PR TITLE
Add CoFold wrapper

### PR DESCRIPTION
**Pull request by bgruening**
@smithcr @TorHou ready for review!

### CODE REVIEW

File: rna_tools/cofold/cofold.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<tool id="cofold" name="Cofold" version="0.0.1">
+    <description>An RNA secondary structure prediction method that takes co-transcriptional folding into account</description>
+    <requirements>
+        <requirement type="package" version="0.0.1"></requirement>
+    </requirements>
+    <version_command>CoFold --version</version_command>
```
Comment (smithcr) [inline rna_tools/cofold/cofold.xml]: wrap in CDATA?

File: rna_tools/cofold/cofold.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            <option value="rna_andronescu2007.par">Andronescu 2007</option>
+        </param>
+        <param name="dangling" type="select" label="how to treat dangling end energies" help="(-d)">
+            <option value="0">0: ignore dangling ends</option>
+            <option value="1">1: unpaired bases participate in one dangling end only</option>
+            <option value="2" selected="True">2: unpaired bases participate in all dangling ends</option>
+            <option value="3">3: allow coaxial stacking</option>
+        </param>
+        <conditional name="advancedOptions">
+            <param name="advancedSelector" type="select" label="Advancec Options">
```
Comment (smithcr) [inline rna_tools/cofold/cofold.xml]: Advanced*

File: rna_tools/cofold/cofold.xml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            <param name="infile" value="cofold1.fasta" ftype="fasta"/>
+            <param name="noPS" value="--noPS"/>
+            <output name="tabularFile" file="colfold_results1.tabular" ftype="tabular" />
+        </test>
+        <test>
+            <param name="infile" value="cofold1.fasta" ftype="fasta"/>
+            <param name="noPS" value="--noPS"/>
+            <param name="energy_parameter" value="rna_andronescu2007.par"/>
+            <param name="dangling" value="1"/>
+            <param name="advancedSelector" value="yesn"/>
```
Comment (smithcr) [inline rna_tools/cofold/cofold.xml]: yes*

File: rna_tools/cofold/cofold.xml [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            <output name="tabularFile" file="colfold_results2.tabular" ftype="tabular" />
+        </test>
+        <test>
+            <param name="infile" value="cofold1.fasta" ftype="fasta"/>
+            <param name="noPS" value="False"/>
```
Comment (smithcr) [inline rna_tools/cofold/cofold.xml]: value should = falsevalue = ""

File: rna_tools/cofold/cofold.xml [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    </tests>
+    <help>
+<![CDATA[
+**CoFold**
+
+A tool for prediction of RNA secondary structure that takes co-transcriptional folding into account 
+
+**Input format**
+
+CoFold requires one input file in FASTA file.
```
Comment (smithcr) [inline rna_tools/cofold/cofold.xml]: FASTA format*

**Comment by bgruening** [comment]
Thanks for reviewing. Will merge now.

**Pull request merged**